packages/core/src/devtools/injectDevTools.ts

```diff
@@ -1,22 +1,28 @@
-import { SEQUENCE_PREFIX } from '../constants';
-import { getDebugTree } from './getDebugTree';
+import { SEQUENCE_PREFIX, DEBUG_RESET_CLASSES } from '../constants';
+import { mergeDebugSequence } from './mergeDebugSequence';
 
 export function injectDevTools(document: Document) {
   const window = document.defaultView;
   if (!window || window.__GRIFFEL_DEVTOOLS__) {
     return;
   }
 
-  const devtools: typeof window['__GRIFFEL_DEVTOOLS__'] = {
+  const devtools: (typeof window)['__GRIFFEL_DEVTOOLS__'] = {
     getInfo: (element: HTMLElement) => {
-      const rootDebugSequenceHash = Array.from(element.classList).find(className =>
-        className.startsWith(SEQUENCE_PREFIX),
-      );
-      if (rootDebugSequenceHash === undefined) {
-        return undefined;
-      }
+      let rootDebugSequenceHash: string | undefined;
+      let rootResetDebugClassName: string | undefined;
 
-      return getDebugTree(rootDebugSequenceHash);
+      Array.from(element.classList).forEach(className => {
```

Comment:
```suggestion
      for (const className of element.classList) {
```

nit `Array.from()` does a full iteration over an iterator AFAIR

Reply:
https://github.com/microsoft/griffel/pull/636/commits/815f642e197c428ed514edaa871497d1c2428e5e